### ISSUE TITLE
Incorrect mapping of KRName IVD Collection

### ISSUE THREAD

**Issue by barro350** [adobe-fonts/source-han-sans#276]
### Description

In [SourceHanSans_KR_sequences.txt](https://github.com/adobe-fonts/source-han-sans/blob/master/SourceHanSans_KR_sequences.txt#L30-L33), IVSs for 請 `U+8ACB` are incorrectly mapped to the CIDs of 誠 `U+8AA0`:

```plaintext
8AA0 E0104; KRName; CID+37864
8AA0 E0105; KRName; CID+62792
8ACB E0104; KRName; CID+62792
8ACB E0105; KRName; CID+37864
```

This causes `<U+5ABA, U+E0104>` and `<U+5ABA, U+E0105>` to be rendered as if they were `<U+8AA0, U+E0105>` and `<U+8AA0, U+E0104>`, respectively:

![image](https://user-images.githubusercontent.com/69441424/96346276-bd9a0c00-10d5-11eb-9518-8572dbcce164.png)

**Comment by punchcutter** [comment]
Adding for reference, from https://www.unicode.org/ivd/data/2017-12-12/IVD_Charts_KRName.pdf
<img width="522" alt="KRName" src="https://user-images.githubusercontent.com/11257204/96503264-da366100-1207-11eb-9b34-76990b96c995.png">

**Comment by punchcutter** [comment]
Sorry for the delay. This has been fixed for the next release.

**Comment by punchcutter** [comment]
Fixed in 2.003
<img width="269" alt="276" src="https://user-images.githubusercontent.com/11257204/113973651-9b5c9c00-9877-11eb-8313-e31b75ce32c3.png">
